stl/src/filesystem.cpp

```diff
@@ -761,13 +762,37 @@ _Success_(return == __std_win_error::_Success) __std_win_error
     return __std_win_error{GetLastError()};
 }
 
+static DWORD WINAPI _Stl_GetTempPath2W(
+    _In_ DWORD nBufferLength, _Out_writes_to_opt_(nBufferLength, return +1) LPWSTR lpBuffer) {
+
+    decltype(&::GetTempPath2W) _PfGetTempPath2W;
+    {
+        static _STD atomic<decltype(&::GetTempPath2W)> _Static = nullptr;
+
+        _PfGetTempPath2W = _Static.load(_STD memory_order_relaxed);
+        if (!_PfGetTempPath2W) {
+            const auto _Kernel32 = GetModuleHandleW(L"kernel32.dll");

```

Comment:
If we're going to globally qualify `::GetProcAddress()` etc., we should do the same for `GetModuleHandleW()`.